### ISSUE TITLE
pixelRatio value above 1 does not work on echart v5.0.1

### ISSUE THREAD

**Issue by Blakey** [apache/echarts#14179]
### Version
5.0.1

### Steps to reproduce
try the following option in the online editor and click download button:

```
option = {
    xAxis: {
        type: 'category',
        data: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
    },
    yAxis: {
        type: 'value'
    },
    series: [{
        data: [150, 230, 224, 218, 135, 147, 260],
        type: 'line'
    }],
    toolbox: {
        feature: {
            saveAsImage: {
                pixelRatio:2
            }
        }
    }
};
```

### What is expected?
Download image in higher resolution

### What is actually happening?
You will see that download with pixelRatio 1 does work but value 2 causes an error as seen on the browser console logs.

---
Upgraded from v4 to v5 and everything we use works except this.

**Comment by Blakey** [comment]
I have since noticed that this issue is related to Canvas render mode. Is the only way to increase the resolution for downloads to use SVG rather than canvas ?

**Comment by plainheart** [comment]
Hi, we've fixed this issue in 5.0.2 which is coming soon. You can try the RC version from GitHub releases before publishing the formal version to NPM.
https://github.com/apache/echarts/releases/tag/5.0.2-rc.1
